Round Robin (#210)
jaybee84/ml-in-rd: content/03.combining-datasets.md

When small sample sizes compromise an ML model's performance, then two approaches can be taken to manage sparsity, variance, and collinearity: 1) increase the number of samples, 2) improve the quality of samples.
In the first approach, appropriate training, evaluation, and held-out validation sets could be constructed by combining multiple rare disease cohorts (Figure {@fig:1}a, Box 1).
When combining datasets, special attention should be directed towards _data harmonization_ since data collection methods can differ from cohort to cohort.
Without careful selection of aggregation methods, one may introduce variability into the combined dataset that can negatively impact the ML model's ability to learn or detect meaningful signal.
Steps such as reprocessing the data using a single pipeline, using batch correction methods [@doi:10.1093/biostatistics/kxj037; @doi:10.1093/nar/gku864], and normalizing raw values [@doi:10.1186/gb-2010-11-3-r25] may be necessary to mitigate unwanted variability. (Figure {@fig:1}a)

> In the second approach, small but robustly generated datasets can enhance the performance of ML models.

Reviewer: Going to throw out some substitutes for robustly here: intentionally, thoughtfully, appropriately

> In the second approach, small but robustly generated datasets can enhance the performance of ML models.

Author: ```suggestion
In the second approach, small but thoughtfully generated datasets can enhance the performance of ML models.
```

> Specifically, improving the labeling of data is critical in accounting for sparsity and variance in the data. 

Reviewer: I'm not sure that my suggestion retains the same meaning. I think we're using "labeling of data" to mean overall annotation or phenotyping, not just the labels one might want to predict.

```suggestion
Specifically, improving data labels is critical in accounting for sparsity and variance in the data. 
```

Author: @jaclyn-taroni how about this? 

```suggestion
One way to improve the quality of a dataset is to improve the accuracy of clinical annotations or phenotypes for each sample in the dataset. 
```

Reviewer: Sounds good! I think it will also dovetail nicely into a question I had about making this section (post-removal of the anecdotal example) sufficiently different from what we talk about in the outlook over on #211

Reviewer: I'm going to commit this change in the interest of getting this merged and resolving conflicts.

> Specifically, improving the labeling of data is critical in accounting for sparsity and variance in the data. 

Author: ```suggestion
One way to improve the quality of a dataset is to improve the accuracy of clinical annotations or phenotypes for each sample in the dataset. 
```
